perun-openapi/openapi.yml

```diff
@@ -2384,6 +2400,127 @@ paths:
         default:
           $ref: '#/components/responses/ExceptionResponse'
 
+  /json/authzResolver/getGroupRoleNames:
+    get:
+      tags:
+        - AuthzResolver
+      operationId: getGroupRoleNames
+      summary: |
+        Returns list of group's role names. Perun system uses role names in the upper case format. However,
+        for now, they are converted to the lower case format because of the compatibility with external systems.
+      responses:
+        '200':
+          $ref: '#/components/responses/ListOfStringsResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
+
+  /urlinjsonout/authzResolver/getGroupRoles:
+    get:
+      tags:
+        - AuthzResolver
+      operationId: getGroupRoles
+      summary: Returns all roles as an AuthzRoles object for a given group.
+      parameters:
+        - $ref: '#/components/parameters/groupId'
```

Comment:
This might sound a bit dumb but this method actually takes a `groupId` param. `#/components/parameters/groupId` refers to a param with name `group`.